docs/query_tool.md

```diff
@@ -126,3 +126,10 @@ This number will be the same across rows corresponding to the same subject.
 Example:
 
 {{ read_table('./repos/neurobagel_examples/query-tool-results/participant-level-results.tsv') }}
+
+#### Participant-level results in aggregate mode
```

Comment:
🍒: to make it super easy to find in the nav for folks who have used the default env var values and are confused about the `protected` everywhere (👀 https://github.com/neurobagel/api/issues/200):
```suggestion
#### `protected` participant-level results in aggregate mode
```